packages/utils/package.json

```diff
@@ -2,8 +2,9 @@
   "name": "@quality-metrics/utils",
   "version": "0.0.1",
   "dependencies": {
+    "@quality-metrics/models": "^0.0.1",
     "chalk": "^5.3.0",
     "cliui": "^8.0.1",
-    "@quality-metrics/models": "^0.0.1"
+    "type-fest": "^4.3.1"
```

Comment:
Is it worth to pull in a dependency for typing the `package.json`? I don't really use this lib so I just ask...

Reply:
Probably not, since we only need `{ name?: string, version?: string }`. I'll remove it.